src/pages/messaging/agent-chat-api/v3.5/index.mdx

```diff
@@ -505,7 +505,8 @@ You can pass only one of the following values at a time: `exists`, `values` or `
 `<survey>` contains the following fields:
 
 - `type` (`string`) - allowed values: `pre_chat`, `post_chat`
-- `answer_id` (`string`)
```

Comment:
@MaciejSzaflik :)

Reply:
https://github.com/livechat/livechat-public-docs/pull/1179#discussion_r909263489
same as above, filters are the same in 3.4 and in 3.5

Reply:
as above, should be fine now :)